PurchasesCoreSwift/Logging/Strings/AttributionStrings.swift

```diff
@@ -8,33 +8,60 @@
 
 import Foundation
 
+// swiftlint:disable identifier_name
 @objc(RCAttributionStrings) public class AttributionStrings: NSObject {
-    @objc public var appsflyer_id_deprecated: String { "The parameter key rc_appsflyer_id is deprecated. Pass networkUserId to addAttribution instead." }
```

Comment:
I'm pretty OK with single line statements like this. I think we should consider disabling the check for this.

Reply:
I reverted and reapplied --autocorrect and it didn't change it so I think we are fine, it was probably just me trying to avoid the 120 char limit. I reverted the lines that can be one line 👍